manual/src/refman/extensions/effects.etex

```diff
@@ -305,9 +241,6 @@ offered and immediately resume the current task with the value being offered.
 If not, we make the current task the waiting exchanger, and run the next task
 from the scheduler queue.
 
-Note that this scheduler code is not perfect -- it can leak resources. We shall
-explain and fix this in the next section~\ref{s:effects-discontinue}.
-
```

Comment:
What was the issue with this paragraph?

Reply:
This paragraph seemed out of place and not very useful. Nothing is "perfect." We say it leaks resources but just point to the next section. It didn't feel right to say that there was a problem and then leave it at that. I felt that the next section directly jumps into the problem, and proposes the solution as well. Unless we feel that users may copy-paste this code without reading the next section, I felt that it was better not to have this paragraph. Thoughts?

Reply:
If there were a blatant drop of a continuation, I think it would be good to point out that this is an error, which will be explained in the next section. But since the leak in the example code is not that blatant, I agree that this forward reference is not really useful.

Reply:
Thanks. I'll resolve this conversation.